app/controllers/provider/admin/applications_controller.rb

```diff
@@ -27,6 +27,7 @@ def index; end
   def show
     @service = @cinstance.service
     @utilization = @cinstance.backend_object.utilization(@service.metrics)
+    activate_menu(:serviceadmin)
```

Comment:
Because that would make Audience > Applications > Listing, before clicking on an actual Application, change the context menu to Products, which I understand is not what is expected. It would also break the lateral menu, since it is an unexpected behaviour. We want to display Products as context menu just when we are inside the application, or at least that's what I considered.

Reply:
Thing is, `Provider::Admin::ApplicationsController` handles the Audience context, whereas `Api::ApplicationsController` handles the Product context.
If we look at the vert menu we can see the difference:
- Audience: Accounts, Applications, Billing, Dev Portal...
- Products: Overview, Analytics, Integration...

So for this particular #show method the problem may simply be that the vert menu should not show the product name. I don't know about the other cases, if any.

Reply:
Well, I think it might be the same behaviour as when you are in Products > Integration and check your backend. You're redirected to the specific backend overview at Backends so you're right, not showing the product name could be the solution here.